Splitwise_cli_2.py

```diff
@@ -0,0 +1,132 @@
+
+
+def main():
+    import sys
+    import argparse
```

Comment:
Any reason you have imports inside method? What's the difference between this being on top vs inside method?